subprojects/gradle/cd/src/gradleTest/kotlin/com/avito/ci/steps/CustomTaskStepTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.avito.ci.steps
+
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.TestResult
+import com.avito.test.gradle.gradlew
+import com.avito.test.gradle.plugin.plugins
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+import java.nio.file.Path
+
+internal class CustomTaskStepTest {
```

Comment:
add case when we add multiple `customTask`

Reply:
Added this case - `customTask - executes tasks - multiple steps`